pwndbg/gdblib/regs.py

```diff
@@ -133,17 +136,17 @@ def all(self):
             elif isinstance(regset, (list, tuple)):
                 retval.extend(regset)
             elif isinstance(regset, dict):
-                retval.extend(regset.keys())
+                retval.extend(list(regset.keys()))
```

Comment:
```suggestion
                retval.extend(regset.keys())
```

not sure what tool generated this but its stupid: .list.extend should take an interable, not a list and dict.keys returns an interable

Reply:
fix!